legion/engine/physics/systems/physicssystem.hpp

```diff
@@ -303,7 +304,18 @@ namespace legion::physics
 
                     if (isTriggerInvolved)
                     {
+
+                        //TODO:(algo-ryth-mix,Developer-The-Great):
+                        //TODO:(cont.) The second paramenter here is supposed to be the delta-time of the physics-system
+                        //TODO:(cont.) A: do we need that? and B: it currently is not.
+                        //
+
+                        //notify the event-bus
+                        raiseEvent<TriggerEvent>(m,1.0f);
                         //notify both the trigger and triggerer
+                        //TODO:(Developer-The-Great): the triggerer and trigger should probably received this event
+                        //TODO:(cont.) through the event bus, we should probably create a filterable system here to
+                        //TODO:(cont.) uniquely identify involved objects and then redirect only required messages
```

Comment:
the physics manifold contains the physicsComponent, which has a reference to the colliding entities. So this could be a way of uniquely identifying involved objects.